### PR TITLE
Relese 1.20 performance test

### PR DESCRIPTION
**Pull request by Shuayb1**
## Description

Performance updates for 1.20

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [x] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Review by AndreaM12345 — APPROVED** [review]

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 1]
```diff
@@ -101,25 +101,27 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-### Latency for Istio 1.19
+### Latency for Istio 1.20
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
 proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
```
Comment (ericvn) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:107]: I think we want this to read the specific version and not the current release:
```suggestion
proxy. In the default configuration of Istio 1.20 (i.e. Istio with telemetry v2),
```

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 2]
```diff
@@ -101,25 +101,27 @@ is busy handling the request, the worker won't start handling the next request
...
-the two proxies add about 1.31 ms and 1.58 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+the two proxies add about 0.228 ms and 0.298 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
 We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
-for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
+for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 2,4,8,16,32,64 client connections, 2 proxy workers and mutual TLS enabled.
```
Comment (ericvn) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:111]: Can we add some documentation to the graphs or page to better differentiate the two graphs. I think a title would suffice. It's hard to tell just looking at the page that one is 90th percentile and one is 99th percentile without reading the test, etc. If you just look at the pictures, you see two with no titles, having the same horizontal and vertical axis, but different values. We should make it easier to understand the data just looking at the picture.

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 3]
```diff
@@ -101,25 +101,30 @@ is busy handling the request, the worker won't start handling the next request
...
+Note: Different hardware will give different values.
+
+<br><img width="90%" style="display: block; margin: auto;"
+    src="latency-p90-fortio-with-uniform.svg"
     alt="P90 latency vs client connections"
     caption="P90 latency vs client connections"
->}}
+/>
+<p><h2 style="text-align: center;"> P90 latency vs client connections </h2></p><br>
+
```
Comment (ericvn) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:121]: ./en/docs/ops/deployment/performance-and-scalability/index.md:120: MD012 Multiple consecutive blank lines

Further documentation is available for these failures:
 - MD012: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md012---multiple-consecutive-blank-lines:
```suggestion

```

**Comment by ericvn** [comment]
And I'm not sure what the merge conflict is...

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 4]
```diff
@@ -135,4 +134,4 @@ Istio uses the following tools for benchmarking
 
 - [`fortio.org`](https://fortio.org/) - a constant throughput load testing tool.
 - [`nighthawk`](https://github.com/envoyproxy/nighthawk) - a load testing tool based on Envoy.
-- [`isotope`](https://github.com/istio/tools/tree/{{< source_branch_name >}}/isotope) - a synthetic application with configurable topology.
+- [`isotope`](https://github.com/istio/tools/tree/{{< source_branch_name >}}/isotope) - a synthetic application with configurable topology.
```
Comment (ericvn) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:137]: Looks like it didn't like this change:
```
./en/docs/ops/deployment/performance-and-scalability/index.md:136: MD047 File should end with a single newline character

Further documentation is available for these failures:
 - MD047: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md047---file-should-end-with-a-single-newline-character
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**